.ci-scripts/test-ExampleTestProject-with-CLI.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+# Copyright (c) Bosch Software Innovations GmbH 2019.
+#
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v2.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v20.html
+#
+# SPDX-License-Identifier: EPL-2.0
+
+set -ex
```

Comment:
Nit: Set options could be moved to the shebang line.

Reply:
not with the `#!/usr/bin/env bash` shebang

Reply:
Ah, right. Does it really make sense to use the `env` approach with `bash`? According to [this answer](https://unix.stackexchange.com/a/77586/53328) "a bash script that only works on Linux can safely use `#!/bin/bash`".

Reply:
That is not correct. E.g. on my linux there is no file `/bin/bash` and also on mac the env line is necessary (afaik).

Reply:
On Mac is the location depending on the installation method you use, so I would prefer also env tool as the shebang.

Reply:
Good to know, thanks.